src/assets/SingleArticle.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import "./CSS/SingleArticle.css"
+
+export default function SingleArticle() {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsError(false);
+    axios
+      .get(`https://js-be-project.onrender.com/api/articles/${article_id}`)
+      .then((apiResponse) => {
+        setArticle(apiResponse.data.articles[0]);
```

Comment:
We spoke about this but just as a reminder, would be worth coming back to at some point!